test/web-platform-tests/to-upstream/html/browsers/browsing-the-web/navigating-across-documents/fragment-replace.html

```diff
@@ -11,50 +11,47 @@
 setup({ allow_uncaught_exception: false });
```

Comment:
Interesting, this test now fails in Edge and Firefox, but not Chrome. If you are able to do the work of tracking down exactly why, that would be awesome (and we can maybe straighten it out in the HTML spec). But if not that's fine too.